### PR TITLE
libpython: fix create new project (location)

### LINKED ISSUE #3587: [Bug] Creating new project (location) with wxGUI Location wizard fails
**Describe the bug**
Creating new project (location) with wxGUI Location wizard fails.

**To Reproduce**

1. Launch wxGUI
2. Invoke Data catalog root database tree node menu and choose **Create new project (location)** item
3. Create new project (location) with e.g. EPSG 5514

```
Traceback (most recent call last):
  File
"/usr/lib64/grass84/gui/wxpython/datacatalog/tree.py", line
1123, in OnCreateLocation

self.CreateLocation(self.selected_grassdb[0])
  File
"/usr/lib64/grass84/gui/wxpython/datacatalog/tree.py", line
1103, in CreateLocation

grassdatabase, location, mapset =
create_location_interactively(

  File
"/usr/lib64/grass84/gui/wxpython/startup/guiutils.py", line
166, in create_location_interactively

gWizard = LocationWizard(parent=guiparent,
grassdatabase=grassdb)

  File
"/usr/lib64/grass84/gui/wxpython/location_wizard/wizard.py",
line 2515, in __init__

msg = self.OnWizFinished()

  File
"/usr/lib64/grass84/gui/wxpython/location_wizard/wizard.py",
line 2723, in OnWizFinished

grass.create_location(
  File "/usr/lib64/grass84/etc/python/grass/script/core.py",
line 1706, in create_location

return create_project(*args, **kwargs)

TypeError
:
create_project() got an unexpected keyword argument 'dbase'
```

**Expected behavior**
Creating new project (location) with wxGUI Location wizard should work.

**System description (please complete the following information):**

- Operating System: all
- GRASS GIS version 8.4.0dev

```
GRASS nc_spm_08_grass7/climate_2000_2012:~ > python3 -c "import sys, wx; print(sys.version); print(wx.version())"
3.11.8 (main, Mar 26 2024, 07:59:33) [GCC 13.2.1 20240210]
4.2.1 gtk3 (phoenix) wxWidgets 3.2.2.1
```

**Additional context**

Related commit https://github.com/OSGeo/grass/commit/d32ece4ff46828f6a8c702a2e3693fdefa470149.

### PR DESCRIPTION
**Pull request by tmszi**
Fixes #3587.

### CODE REVIEW

**Review by petrasovaa — CHANGES_REQUESTED** [review]
While these changes are fine, I think it should be primarily addressed in `create_location()` to make it backwards compatible, similarly to how `location` gets renamed to '`name`.

**Comment by tmszi** [comment]
> While these changes are fine, I think it should be primarily addressed in `create_location()` to make it backwards compatible, similarly to how `location` gets renamed to '`name`.

Fixed with https://github.com/OSGeo/grass/pull/3588/commits/adaf0d01f85a267568dc31d44a2b34ec01892e39.

File: python/grass/script/core.py [hunk 1]
```diff
@@ -1700,6 +1700,9 @@ def mapsets(search_path=False, env=None):
 
 
 def create_location(*args, **kwargs):
+    if "dbase" in kwargs:
+        kwargs["path"] = kwargs["dbase"]
+        del kwargs["dbase"]
```
Comment (chaedri) [inline python/grass/script/core.py:1705]: Looks great! I tested too. Thank you @tmszi for catching that.

**Review by chaedri — APPROVED** [review]

**Comment by chaedri** [comment]
Looks great! I tested too. Thank you @tmszi for catching that.

**Review by petrasovaa — APPROVED** [review]

**Pull request merged**